blueprintapi/environment.py

```diff
@@ -21,7 +21,7 @@ def __init__(self):
             # allowed hosts.
             metadata_uri = os.environ.get("ECS_CONTAINER_METADATA_URI")
             if metadata_uri:
-                container_metadata = requests.get(metadata_uri).json()
+                container_metadata = requests.get(metadata_uri, timeout=30).json()
```

Comment:
Just curious, what's the timeout for here?

Reply:
`pylint` suggestion, but it's good practice regardless in case something hangs. We don't want an endless loop!

Reply:
30 -> 30s btw